gulpfile.babel.js

```diff
@@ -71,10 +78,10 @@ gulp.task('styles', () => {
   const dest = 'valadoc.org/styles'
 
   return gulp.src(src, { base })
-  .pipe(postcss([
-    cssnext({ browsers })
-  ]))
-  .pipe(gulp.dest(dest))
+    .pipe(postcss([
+      cssnext({ browsers })
```

Comment:
We should upgrade `cssnext` to `postcss-env` as `cssnext` is deprecated. It will also use the browser list for building the css.

Reply:
👍

Reply:
done